README.md

```diff
@@ -64,6 +72,9 @@ If needed static resources like **images**,... can be shared by using symlinks i
 
 ```
 .
+├── proxies
```

Comment:
Why not just put it under the `packages/events-federation-gateway` ?

Reply:
Okay I understand it's not application nor just shared code, just a server. Maybe that's folder structure is ok then.